client/src/components/new-request/stats.vue

```diff
@@ -12,26 +12,36 @@
 <!--            No Wait Time-->
 <!--          </div>-->
 <!--        </div>-->
-        <div class="stats-content-outer py-0">
-          <div class="stats-content-inner-1 text-center">
-            <div class="stats-value h3-lt">{{ priorityWaitTime }}</div>
-            <div class="stats-unit">Hours</div>
+      <v-tooltip bottom nudge-left="56" content-class="bottom-tooltip wait-time-tooltip" transition="fade-transition">
+        <template v-slot:activator="{ on }">
+          <div class="stats-content-outer py-0" v-on="on">
+            <div class="stats-content-inner-1 text-center">
+              <div class="stats-value h3-lt">{{ priorityWaitTime }}</div>
```

Comment:
Is this (and regularWaitTime below) an integer or floating point number? If the latter, you may need a `toFixed()` here to limit the decimals.

Reply:
I really have idea about the workings of the component, i just added a tooltip wrapper. I can look into this tho if you like?

Reply:
No, leave it for now.